airbyte-webapp/src/hooks/services/Feature/types.tsx

```diff
@@ -1,3 +1,8 @@
+/**
+ * FeatureItems are for permanent flags to differentiate features between environments (e.g. Cloud vs. OSS),
+ * workspaces, specific user groups, etc. The corresponding flag should be added to LaunchDarkly.
```

Comment:
> The corresponding flag should be added to LaunchDarkly.

Not entirely sure we should put that in here? For features we currently don't have that PR that makes each feture an individual flag (https://github.com/airbytehq/airbyte/pull/19773) merged yet.

Did you mean to put this on top of the experiments.ts file, where we will need a LD flag for everything?

Reply:
I've just removed it now. I guess it's not necessary to be added to LD (only if you're overriding a default `FeatureItem`), so the explanation about how LD works with `FeatureService` is probably better off in the Notion docs.